api/proto/teleport/legacy/types/types.proto

```diff
@@ -3665,6 +3665,13 @@ message KubernetesResource {
   string Name = 3 [(gogoproto.jsontag) = "name,omitempty"];
   // Verbs are the allowed Kubernetes verbs for the following resource.
   repeated string Verbs = 4 [(gogoproto.jsontag) = "verbs,omitempty"];
+
+  // APIGroup specifies the API group of the resource. Can be '*' for any,
+  // but doesn't support pattern matching.
```

Comment:
why doesn't it support pattern matching?
imagine I want to give access to all cillium crds `^.*cilium.io$`